stdlib/3/builtins.pyi

```diff
@@ -682,28 +682,28 @@ class set(MutableSet[_T], Generic[_T]):
     def __contains__(self, o: object) -> bool: ...
     def __iter__(self) -> Iterator[_T]: ...
     def __str__(self) -> str: ...
-    def __and__(self, s: AbstractSet[Any]) -> Set[_T]: ...
-    def __iand__(self, s: AbstractSet[Any]) -> Set[_T]: ...
+    def __and__(self, s: AbstractSet[object]) -> Set[_T]: ...
+    def __iand__(self, s: AbstractSet[object]) -> Set[_T]: ...
     def __or__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
     def __ior__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
-    def __sub__(self, s: AbstractSet[Any]) -> Set[_T]: ...
-    def __isub__(self, s: AbstractSet[Any]) -> Set[_T]: ...
+    def __sub__(self, s: AbstractSet[object]) -> Set[_T]: ...
+    def __isub__(self, s: AbstractSet[object]) -> Set[_T]: ...
     def __xor__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
     def __ixor__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
-    def __le__(self, s: AbstractSet[Any]) -> bool: ...
-    def __lt__(self, s: AbstractSet[Any]) -> bool: ...
-    def __ge__(self, s: AbstractSet[Any]) -> bool: ...
-    def __gt__(self, s: AbstractSet[Any]) -> bool: ...
+    def __le__(self, s: AbstractSet[object]) -> bool: ...
+    def __lt__(self, s: AbstractSet[object]) -> bool: ...
+    def __ge__(self, s: AbstractSet[object]) -> bool: ...
+    def __gt__(self, s: AbstractSet[object]) -> bool: ...
     # TODO more set operations
 
 class frozenset(AbstractSet[_T], Generic[_T]):
     def __init__(self, iterable: Iterable[_T] = ...) -> None: ...
     def copy(self) -> FrozenSet[_T]: ...
-    def difference(self, *s: Iterable[Any]) -> FrozenSet[_T]: ...
-    def intersection(self, *s: Iterable[Any]) -> FrozenSet[_T]: ...
+    def difference(self, *s: Iterable[object]) -> FrozenSet[_T]: ...
```

Comment:
That makes sense from the perspective of minimizing false positives. It seems a shame that mypy won't tell you if you're mixing sets of different types in your operations (which is surely often a bug), but we should generally err on the side of reducing false positives.